### DISCUSSION TITLE
Ja3 hash notice log not generating

### DISCUSSION THREAD

**Discussion by Jackson-Pollock** [Security-Onion-Solutions/securityonion#8248] [Unsupported Versions]
I created custom zeek script to detect Ja3 hash, It was verified on https://try.bro.org/ and it was generating notice.log. but same script doesn't work in Security Onion.

**Comment by dougburks** [comment]
Did you follow the steps at https://docs.securityonion.net/en/2.3/zeek.html#custom-scripts?

**Reply by Jackson-Pollock** [reply]
yes, custom scripts other than Ja3 are working correctly.

**Reply by Jackson-Pollock** [reply]
Also I tried same script in local zeek installed in ubuntu and it was working, but not in SO.

**Reply by dougburks** [reply]
Per https://docs.securityonion.net/en/2.3/zeek.html#custom-scripts, have you checked `/nsm/zeek/logs/current/loaded_scripts.log` to see if it's being loaded?
